Spec/RealtimeClientConnection.swift

```diff
@@ -4247,6 +4247,7 @@ class RealtimeClientConnection: QuickSpec {
                 // https://github.com/ably/wiki/issues/22
                 it("should encode and decode fixture messages as expected") {
                     let options = AblyTests.commonAppSetup()
+                    options.useBinaryProtocol = false
```

Comment:
This started failing because of 4ff2639, right?

Reply:
Correct, same for https://github.com/ably/ably-ios/pull/757/files#diff-ebbc11fa8ab1c043fe6ed321bbbe72bbR999